packages/x-tree-view/src/internals/plugins/useTreeViewSelection/useTreeViewSelection.types.ts

```diff
@@ -26,32 +26,43 @@ export interface UseTreeViewSelectionParameters<Multiple extends boolean | undef
    * When `multiSelect` is true this takes an array of strings; when false (default) a string.
    * @default []
    */
-  defaultSelected?: TreeViewSelectionValue<Multiple>;
+  defaultSelectedNodes?: TreeViewSelectionValue<Multiple>;
   /**
    * Selected node ids. (Controlled)
    * When `multiSelect` is true this takes an array of strings; when false (default) a string.
    */
-  selected?: TreeViewSelectionValue<Multiple>;
+  selectedNodes?: TreeViewSelectionValue<Multiple>;
   /**
    * If true `ctrl` and `shift` will trigger multiselect.
    * @default false
    */
   multiSelect?: Multiple;
   /**
-   * Callback fired when tree items are selected/unselected.
+   * Callback fired when tree items are selected/deselected.
```

Comment:
See https://english.stackexchange.com/questions/18465/unselect-or-deselect#:~:text=The%20NOAD%20defines%20deselect%20as,selected%20and%20isn't%20anymore.